### ISSUE TITLE
Jackson ObjectMapper best practice

### ISSUE THREAD

**Issue by iyourshaw** [usdot-jpo-ode/jpo-geojsonconverter#19]
Recommendation to follow the best practices for using Jackson's ObjectMapper.  It is a heavy duty object that is thread safe, so it is recommended to create one instance of it and reuse it:

See the guidelines here:

[Jackson Docs: Performance](https://github.com/FasterXML/jackson-docs/wiki/Presentation:-Jackson-Performance)

This could be done by changing the `DateJsonMapper` which creates a new instance of ObjectMapper with each call to `getInstance`:

```java
public class DateJsonMapper {

    public static ObjectMapper getInstance() {
        ObjectMapper objectMapper = new ObjectMapper();
        objectMapper.registerModule(new JavaTimeModule());
        objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
        objectMapper.setSerializationInclusion(Include.NON_NULL);
        return objectMapper;
    }
}
```

To a singleton pattern like:
```java
public class DateJsonMapper {

    final static ObjectMapper objectMapper;
    
    static {
        objectMapper = new ObjectMapper();
        objectMapper.registerModule(new JavaTimeModule());
        objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
        objectMapper.setSerializationInclusion(Include.NON_NULL);
    }

    public static ObjectMapper getInstance() {
        return objectMapper;
    }
}
```

**Comment by drewjj** [comment]
This makes a lot of sense. The jpo-ode creates many of these objects within it which is why I think we chose to do it this way. Definitely worthwhile implementing it correctly.

**Comment by iyourshaw** [comment]
Fixed in PR #18
